pkg/commands/drift/drift.go

```diff
@@ -129,34 +159,58 @@ func actualGCPIAM(
 		return nil, fmt.Errorf("failed to initialize iam client: %w", err)
 	}
 
-	m := make(map[string]*iam.AssetIAM)
-	oIAM, err := client.OrganizationIAM(ctx, organizationID)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get organization IAM for ID '%s': %w", organizationID, err)
-	}
-	for _, i := range oIAM {
-		m[URI(i, organizationID, foldersByID, projectsByID)] = i
-	}
-	for _, f := range foldersByID {
-		fIAM, err := client.FolderIAM(ctx, f.ID)
+	p := pool.New().
```

Comment:
If we used NewWithResults / ResultPool, could we avoid the need for channels?

Reply:
Done